api/create_materialized_view.md

```diff
@@ -22,8 +22,12 @@ GROUP BY time_bucket( <const_value>, <partition_col_of_hypertable> ),
 [HAVING ...]
 ```
 
-The continuous aggregate view is automatically refreshed unless `WITH NO DATA`
-is given. This setting defaults to `WITH DATA`. For more information, see
+The continuous aggregate view defaults to `WITH DATA`. This means that when the
+view is created, it refreshes using all the current data in its underlying
+hypertable. This occurs once when the view is created. If you want the view to
+be refreshed regularly, you can use a refresh policy. If you do not want the
+view to update at all, even when it is first created, use the `WITH NO DATA`
```

Comment:
```suggestion
view to update when it is first created, use the `WITH NO DATA`
```

From my understanding, it would still update according to the refresh policy if you set one